opsi/modules/nt.py

```diff
@@ -95,42 +102,33 @@ class Inputs:
         val: AnyType
 
     def run(self, inputs):
-        self.table[self.settings.key] = inputs.val
+        if inputs.val is None:  # Do not write None values to NT
+            pass
```

Comment:
pass = do nothing
I think you meant return

Reply:
Since there's a ```return``` below the if/else statement below, doing nothing is correct in this case. I agree that the way it is written is unclear though, I'll improve it when I get home tonight.